lib/screens/choose_best_answer.dart

```diff
@@ -1,40 +1,113 @@
-                builder: (context) => Rating(
-                  questionId: widget.questionId,
-                  answerId: widget.answerId,
+      body: SingleChildScrollView(
+        child: Container(
+          constraints: BoxConstraints(
+            minHeight: MediaQuery.of(context).size.height - 269.0,
+          ),
+          margin: EdgeInsets.fromLTRB(10, 10, 10, 0),
+          padding: EdgeInsets.fromLTRB(3, 0, 3, 0),
+          decoration: BoxDecoration(
+            color: Colors.grey[300],
+            borderRadius: BorderRadius.all(
+              Radius.circular(20.0),
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.fromLTRB(20, 0, 20, 10),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  "Answers",
+                  style: TextStyle(
+                    fontSize: 20.0,
+                    fontWeight: FontWeight.bold,
+                    decoration: TextDecoration.underline,
+                    color: Color(0xFF00ADB5),
+                  ),
                 ),
-              ), //archive question
-            );
-          },
-          child: Text('Best Answer'),
+                SizedBox(
+                  height: 20.0,
+                ),
+                for (UserAnswerModel userAnswer in widget.answers)
+                  GestureDetector(
+                    onTap: () => bestAnswerSelector(
+                      widget.answers.indexOf(userAnswer),
+                    ),
+                    child: Container(
+                      decoration: BoxDecoration(
+                        color: widget.answers.indexOf(userAnswer) ==
+                                bestAnswerIndex
+                            ? Color(0xFF00ADB5)
+                            : Colors.transparent,
+                        borderRadius: BorderRadius.all(
+                          Radius.circular(20.0),
+                        ),
+                      ),
+                      child: MoreDetailsView(
+                        isAnswer: true,
+                        userAnswerModel: userAnswer,
+                        isCurrUser: false,
+                      ),
+                    ),
+                  ),
+              ],
+            ),
+          ),
+        ),
+      ),
+      bottomNavigationBar: BottomAppBar(
+        color: Colors.transparent,
+        elevation: 0,
+        // For demo purposes this is true, however will be changed to != later
+        child: EurekaRoundedButton(
+          onPressed: bestAnswerIndex == -1
+              ? null
+              : () => Navigator.push(
+                    context,
+                    MaterialPageRoute(
+                      builder: (context) => Rating(
+                        questionId: widget.questionId,
+                        answerId: widget.answers[bestAnswerIndex].answer.id,
+                      ),
```

Comment:
Do we have to check if userid matches with answer.userid , so we don't rate ourselves? Or is it for the next task?